include/OperatorsForMesons.hpp

```diff
@@ -80,7 +95,7 @@ class OperatorFactory {
   /** @TODO comment private members */
   const OperatorLookup operator_lookuptable;
   const ssize_t Lt, Lx, Ly, Lz;
-  const ssize_t nb_ev, dilE;
+  const ssize_t nb_ev, dilE=0;
```

Comment:
This is really bad, just pass ```dilE = 0``` in the constructor if you need this!

Reply:
To be honest, I'm surprised that this even compiles..

Reply:
So this is allowed since C++11, although inconsistent with the style used elsewhere. Could you remove ```dilE=0``` please?